nodejs/lancedb/table.ts

```diff
@@ -456,10 +456,12 @@ export abstract class Table {
 
 export class LocalTable extends Table {
   private readonly inner: _NativeTable;
+  private readonly schema_: Schema;
 
-  constructor(inner: _NativeTable) {
+  constructor(inner: _NativeTable, schema: Schema) {
     super();
     this.inner = inner;
+    this.schema_ = schema;
   }
```

Comment:
Are you caching this to avoid the async call?  I'm not sure it will work.  The reason that `schema()` is asynchronous is because it is possible for the table to have been changed (by other processes) and we want to respect the `read_consistency_interval` and detect this change automatically.

Reply:
Fixed, cache only the registered embedding functions